lib/metanorma/standoc.rb

```diff
@@ -1,7 +1,10 @@
 require_relative "./standoc/processor"
+require_relative "./standoc/requirement"
 
 module Metanorma
   module Standoc
-
+    Requirements = {
+      :latexml => LatexmlRequirement.new
```

Comment:
Can we use the symbol syntax for hash keys when possible?

`:latexml => LatexmlRequirement.new` =>
`latexml: LatexmlRequirement.new`

Reply:
Doe